tailwind.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
+
```

Comment:
@AkshataKatwal16 we are not going to use tailwind css please remove this file and its dependencies